compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/BTypeSymbol.java

```diff
@@ -32,6 +32,7 @@ public class BTypeSymbol extends BSymbol implements TypeSymbol {
 
     public boolean isTypeParamResolved;
     public BTypeSymbol typeParamTSymbol;
+    public BVarSymbol annotations;
```

Comment:
As discussed, the closure map is only required for the runtime. So we need to check and move this to somewhere more appropriate. Let's do this separately.

Reply:
Do we have an issue for this? Or we can create one issue with a checklist for all the pending items.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/39346